ui/documents/semantic-ui/js/documents/search/ResultsListItem.jsx

```diff
@@ -182,22 +182,38 @@ export const ResultsListItemComponent = ({
 
   const rights = _get(result, "metadata.rights");
 
-  const descriptionStripped = _get(
-    result,
-    "metadata.abstract[0].value",
-    i18next.t("No description")
-  );
+  let abstract = _get(result, "metadata.abstract", []);
+
+  if (abstract.length === 0) {
+    abstract = i18next.t("No description");
+  } else {
+    abstract =
```

Comment:
Use: https://github.com/oarepo/oarepo-ui/blob/main/oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/forms/components/I18nString/I18nString.jsx#L5 to keep this logic in a single place

Reply:
@mirekys Hey, I don't think we can use this component for this purpose, because abstract is not in shape {lang:text}, but rather {lang:lang, value:text}

Reply:
Ahh ok i must have overlooked that, it's fine then